src/hw1/bash_commands.py

```diff
@@ -0,0 +1,35 @@
+def head(file, strings=10):
+    for i in range(strings):
+        print(file.readline(), end="")
+
+
+def tail(file, strings=10):
+    lines = [line for line in file]
+    lines.reverse()
+    for i in range(strings):
+        print(lines[i], end="")
+
+
+def wc(file):
+    wordCount = 0
+    symbolCount = 0
+    lines = [line for line in file]
+    lineCount = len(lines)
+    for line in lines:
+        words = line.split(" ")
+        if "\n" in words:
```

Comment:
Can `\n` be in words, if they come from lines, which are split by `\n` by default?